sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -194,7 +194,7 @@ def get_certificate_version(self, certificate_name, version, **kwargs):
 
     @distributed_trace
     def begin_delete_certificate(self, certificate_name, **kwargs):
-        # type: (str, **Any) -> DeletedCertificate
+        # type: (str, **Any) -> LROPoller
```

Comment:
Is there no way to specify in the type hints that it's an LROPoller with a specific type of value?

Reply:
We could type `LROPoller` as generic (requires a change in azure-core) but doing that in the right way is beyond my present mypy knowledge and may not be a good idea in general. The docstring at least does what you'd like.